docs/onboarding/02-digital-twin-definition-language.md

```diff
@@ -51,3 +53,4 @@ The following exercises involve creating and modifying models using DTDL. The mo
 - How often should a new version of the model be uploaded? Should each deployment create new versions?
 - What is the appropriate use of Property vs Telemetry?
 - When are relationships necessary? What capability do they introduce from a query standpoint?
+- Inherit properties (for reusing common attributes) could be avoided by using marker tags. 

```

Comment:
This is a great point and thing to consider. What if we slightly adjust the verbiage to make it a question, e.g.,:
- How can we leverage marker tags when there is a need to reuse common attributes?